src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -1103,18 +1118,27 @@ export default class DefaultAudioVideoController
       this.meetingSessionContext.videoUplinkBandwidthPolicy &&
       !this.meetingSessionContext.enableSimulcast
     ) {
-      const oldMaxBandwidth = this.meetingSessionContext.videoUplinkBandwidthPolicy.maxBandwidthKbps();
-
-      this.meetingSessionContext.videoUplinkBandwidthPolicy.setHasBandwidthPriority(
-        hasBandwidthPriority
-      );
-      const newMaxBandwidth = this.meetingSessionContext.videoUplinkBandwidthPolicy.maxBandwidthKbps();
+      if (this.useUpdateTransceiverController()) {
+        this.meetingSessionContext.videoUplinkBandwidthPolicy.setHasBandwidthPriority(
+          hasBandwidthPriority
+        );
+        await this.meetingSessionContext.videoUplinkBandwidthPolicy.updateTransceiverController(
+          this.meetingSessionContext.transceiverController
+        );
+      } else {
```

Comment:
Maybe do this as a visually separate block with early return. That has the useful property of not touching the blame for the existing lines.

Reply:
Updated.